dissect/hypervisor/disk/qcow2.py

```diff
@@ -117,6 +117,7 @@ def __init__(self, fh, data_file=None, backing_file=None):
                 self.backing_file = backing_file
 
         super().__init__(self.header.size)
+        self.l2_table = lru_cache(128)(self.l2_table)
```

Comment:
```suggestion
        self.l2_table = lru_cache(128)(self.l2_table)

        super().__init__(self.header.size)
```
I'd say that `super()` has more precedence for being at the bottom here.